proto/minder/v1/minder.proto

```diff
@@ -2290,33 +2332,78 @@ message Profile {
         // id is optional and use for updates to match upserts as well as read operations. It is ignored for creates.
         string id = 1;
         // entity is the entity to select.
-        string entity = 2;
+        string entity = 2 [
```

Comment:
We should have called this entity_type, huh?

Reply:
Yes, that's one of the few examples I was talking about yesterday 😃